bigtop_toolchain/bin/puppetize.sh

```diff
@@ -26,7 +26,7 @@ case ${ID}-${VERSION_ID} in
       dnf -y check-update
       dnf -y install hostname findutils curl sudo unzip wget puppet puppetlabs-stdlib
         ;;
-    ubuntu-16.04)
+    ubuntu-1[6,8].04)
         apt-get update
         apt-get -y install wget curl sudo unzip puppet software-properties-common puppet-module-puppetlabs-apt puppet-module-puppetlabs-stdlib
```

Comment:
On ubuntu-18.04 two extra packages need to be installed to get successful bigtop build:
rsync, dh-python

Reply:
Hmm, sounds strange. I've builded bigtop and some packages (hadoop, hive, hbase,  and zookeeper) without these dependencies.

But ok, i'll add your opinion as a patch.

I also have small troubles with systemd on ubuntu-18.04, standard "sysvinit compatibility" doesn't work properly. In the current initd scripts we creating lock directories, pid files. Systemd expect than file have same pair owner-group as a main process, but it don't.

I wrote new template for systemd services, but doesn't debugged it yet.

Give me a couple of days for this patches :)